plugins/tff_backend/tff_backend_plugin.py

```diff
@@ -56,6 +56,7 @@ def get_handlers(self, auth):
         yield Handler(url='/testing/agreements', handler=AgreementsTestingPageHandler)
         yield Handler(url='/refresh', handler=RefreshHandler)
         yield Handler(url='/refresh/callback', handler=RefreshCallbackHandler)
+        yield Handler(url='/apple', handler=AppleReviewQrHandler)
```

Comment:
can u use /testing/apple/qr

Reply:
Apple needs to use this so not sure why /testing is needed

Reply:
Don't like that it only says apple ...